helm/defectdojo/values.yaml

```diff
@@ -66,7 +70,7 @@ django:
         cpu: 100m
         memory: 128Mi
   nodeSelector: {}
-  replicas: 1
+  replicas: 4
```

Comment:
that's a lot of replicas by default; this will be heavy for people deploying on minikube. why not overriding this parameter if you wish to change the value? the kubernetes.md already gives hint for production deployment with replicas.

Reply:
Ignore this, I was was using same repo after creating PR

Reply:
Ignore this, I was using that in prod, probably ended inside of PR.